.travis.yml

```diff
@@ -13,24 +13,15 @@ matrix:
   - os: osx
     env: PYTHON_VERSION="3.7" DEPS="numpy geos"
 
-before_install:
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
-      wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
-    elif  [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
-      wget https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -O miniconda.sh;
-    fi;
-
 install:
-  - bash miniconda.sh -b -p $HOME/miniconda
-  - export PATH="$HOME/miniconda:$HOME/miniconda/bin:$PATH"
-  - source $HOME/miniconda/etc/profile.d/conda.sh
-  - hash -r;
-  - conda update --yes conda;
-  - conda config --set changeps1 no --set restore_free_channel true;
-  - conda config --append channels conda-forge
-  - conda create -n testenv --yes python=$PYTHON_VERSION $DEPS pytest;
-  - conda activate testenv;
-  - conda --version ; python --version ; pip --version;
+  - wget https://repo.anaconda.com/pkgs/misc/conda-execs/conda-latest-$TRAVIS_OS_NAME-64.exe -O conda.exe
+  - chmod +x conda.exe
+  - export CONDA_ALWAYS_YES=1
+  - ./conda.exe config --set changeps1 no --set restore_free_channel true;
+  - ./conda.exe config --append channels conda-forge
```

Comment:
Conda-forge recommends to "add" instead of "append" it (put it at the top of the channel list). Is there a reason to append it here?

Reply:
I always like to be the default channel the highest priority one. But apart from that, no good reason

Reply:
For just numpy and geos, that is probably fine since there cannot be any incompatibility (geos is not built against numpy). But in general it is recommended to have conda-forge first once you have conda-forge packages that are not pure python (because the conda-forge ones are not always binary compatible with defaults, so if you have a binary packages, all its dependent binary packages should also come from conda-forge)

Reply:
Right, I did run into this issues from time to time. Good to now there is actually a recommendation that would help ;)

I switched it to optional for only the newest build, let's see if it works.

Reply:
When changing the --append to --add on the appveyor build, it actually breaks the build. It installs pytest 3.2.2 instead of 5.2